src/Navigation.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import leafletPip from "@mapbox/leaflet-pip";
+import borderData from "./border";
+import L from "leaflet";
+
+class Navigation extends React.Component {
+    constructor(props) {
+    super(props);
+
+    this.state = {
+      disabled: true,
+      gameStarted: true,
+      direction: "",
+    };
+  }
+
+  handleClick = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      disabled: false,
+      gameStarted: true
+    });
+  };
+
+  
+
+  handleNorthClick = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      disabled: false,
+      gameStarted: true,
+      zoom: 18,
+    });
+  };
```

Comment:
Same deal as in Buttons.js Methods with the same names as these are being sent down as props, and those are what you're actually using in your `onClicks`. Except `handleNorthClick` that you are actually using.